### DISCUSSION TITLE
AsusFX504 - 0.6.9

### DISCUSSION THREAD

**Discussion by RobyRew** [RobyRew/ASUS-FX504GE-Hackintosh_OpenCore#3] [General]
### News / Fixes:
- Updated all kexts and boot files to latest versions.
- Updated the config.plist.
- Changed HDMI output configuration trying to solve some issues.
- Fixed OpenCanopy issues

### Know Issues:
- When finalizing boot, backlight stops working for some reason, I don't know how to fix this, but, a solution can be:
**Press** *fn* + *f11* and it goes on sleep, then press any key to power up again.
If you find a solution to this, please do a "Pull request".

### Old Know Issues:
- Audio on **jack 3.5mm** can be a little buggy, (a fix a can be to adjust the frequencies/volume from the native Apple `Audio MIDI.app`)
- HDMI have issues with some resolutions, I don't know how to fix them.

<hr /><em>This discussion was created from the release <a href='https://github.com/RobyRew/ASUS-FX504GE-Hackintosh_Updated_OpenCore_0.6.8/releases/tag/3.0'>AsusFX504 - 0.6.9</a>.</em>

**Comment by wojko6** [comment]
Are you planning update to 0.7.0?

**Reply by RobyRew** [reply]
Yes, on the next few hours I will release the update, sorry for the waiting.

**Reply by wojko6** [reply]
Thanks!

**Reply by RobyRew** [reply]
> Thanks!

hey, sorry, in the end I won't be able to update the EFI today either, I've been working so far and it's late, I really feel bad, but I'll try tomorrow.

If you need a stable and almost perfect EFI, 0.6.4 is the one that works best of all, in 0.6.5 the backlight has started to fail for some reason, if I update the EFI to 0.7.0 tomorrow I will let you know here

**Reply by wojko6** [reply]
No problem!

**Reply by RobyRew** [reply]
> No problem!

Its out man, very great release

https://github.com/RobyRew/ASUS-FX504GE-Hackintosh_Updated_OpenCore/releases/tag/4.0
